docs/data/packages.json

```diff
@@ -1,12 +1,52 @@
 {
 	"v1": {
-		"libraries": ["fluid-framework", "@fluidframework/counter"],
-		"serviceClients": ["@fluidframework/azure-client", "@fluidframework/tinylicious-client"],
-		"utilities": ["@fluidframework/azure-service-utils"]
+		"Audience": {
+			"Interfaces": ["IMember", "IServiceAudience", "IServiceAudienceEvents"],
+			"Types": ["Myself"]
+		},
+		"Container": {
+			"Enums": ["AttachState"],
+			"Interfaces": ["ContainerSchema", "IFluidContainerEvents", "IConnection"],
+			"Types": [
+				"ConnectionStateType",
+				"ICriticalContainerError",
+				"ContainerErrorTypes",
+				"DriverErrorTypes",
+				"DataObjectClass",
+				"InitialObjects",
+				"LoadableObjectClass",
+				"LoadableObjectClassRecord",
+				"LoadableObjectCtor",
+				"LoadableObjectClass",
+				"SharedObjectClass"
+			]
+		},
+		"DDSes": ["@fluidframework/map"],
+		"serviceClients": ["@fluidframework/azure-client", "@fluidframework/tinylicious-client"]
 	},
 	"v2": {
-		"libraries": ["fluid-framework"],
-		"serviceClients": ["@fluidframework/azure-client", "@fluid-experimental/odsp-client"],
-		"utilities": ["@fluidframework/azure-service-utils"]
+		"Audience": {
+			"Interfaces": ["IMember", "IServiceAudience", "IServiceAudienceEvents"],
+			"Types": ["Myself"]
+		},
+		"Container": {
+			"Enums": ["AttachState"],
+			"Interfaces": ["ContainerSchema", "IFluidContainerEvents", "IConnection"],
+			"Types": [
+				"ConnectionStateType",
+				"ICriticalContainerError",
+				"ContainerErrorTypes",
+				"DriverErrorTypes",
+				"DataObjectClass",
+				"InitialObjects",
+				"LoadableObjectClass",
+				"LoadableObjectClassRecord",
+				"LoadableObjectCtor",
+				"LoadableObjectClass",
+				"SharedObjectClass"
+			]
+		},
+		"DDSes": ["@fluidframework/map", "@fluidframework/tree"],
```

Comment:
I don't think we want to pull the docs from the map and tree packages like this (for v2 and forward). We should just reference members as they are exported by the fluid-framework package. Otherwise we leak the fact that these are separate packages again, which includes some details we intentionally don't expose via `fluid-framework`.

I.e. we should manage the hierarchy directly here, and surface only the key concepts of each DDS.